store/heightsub.go

```diff
@@ -28,27 +28,35 @@ func newHeightSub[H header.Header[H]]() *heightSub[H] {
 	}
 }
 
-// Height reports current height.
-func (hs *heightSub[H]) Height() uint64 {
-	return hs.height.Load()
+func (hs *heightSub[H]) isInited() bool {
```

Comment:
We don't used `heightSub[H].Height` anymore, so updating the method to a more meaningful thing.